command/sub8_missions/sub8/sub_singleton.py

```diff
@@ -312,6 +312,17 @@ def _run_look(self, spotings_req):
 
             yield self.sub.nh.sleep(.5)
 
+class PoseSequenceCommander(object):
```

Comment:
In that case I think the best way to go about it is to put the current move_command in a for loop